docs/geometry.rst

```diff
@@ -71,6 +71,41 @@ which this geometry code can position independently.
 
    .. automethod:: from_h5_file_and_quad_positions
 
+   .. automethod:: to_distortion_array
+
+   .. automethod:: plot_data_fast
+
+   .. automethod:: position_modules_fast
+
+   .. automethod:: inspect
```

Comment:
Shouldn't this also mention the crystfel methods (read/write)?

Reply:
Yes, good point. I'll add them.